services/wallet/transaction.go

```diff
@@ -31,18 +65,19 @@ const (
 )
 
 type PendingTransaction struct {
-	Hash           common.Hash    `json:"hash"`
-	Timestamp      uint64         `json:"timestamp"`
-	Value          bigint.BigInt  `json:"value"`
-	From           common.Address `json:"from"`
-	To             common.Address `json:"to"`
-	Data           string         `json:"data"`
-	Symbol         string         `json:"symbol"`
-	GasPrice       bigint.BigInt  `json:"gasPrice"`
-	GasLimit       bigint.BigInt  `json:"gasLimit"`
-	Type           PendingTrxType `json:"type"`
-	AdditionalData string         `json:"additionalData"`
-	ChainID        uint64         `json:"network_id"`
+	Hash               common.Hash    `json:"hash"`
+	Timestamp          uint64         `json:"timestamp"`
+	Value              bigint.BigInt  `json:"value"`
+	From               common.Address `json:"from"`
+	To                 common.Address `json:"to"`
+	Data               string         `json:"data"`
+	Symbol             string         `json:"symbol"`
+	GasPrice           bigint.BigInt  `json:"gasPrice"`
+	GasLimit           bigint.BigInt  `json:"gasLimit"`
+	Type               PendingTrxType `json:"type"`
+	AdditionalData     string         `json:"additionalData"`
+	ChainID            uint64         `json:"network_id"`
+	MultiTransactionId int64          `json:"multi_transaction_id"`
```

Comment:
```suggestion
	ChainID            uint64         `json:"networkId"`
	MultiTransactionId int64          `json:"multiTransactionId"`
```

Also, probably `MultiTransactionId` should be named `MultiTransactionID`. I don't know if the linter catches this